### DISCUSSION TITLE
Is there a Documentation for this

### DISCUSSION THREAD

**Discussion by SNJnet** [Cosium/matrix-communication-client#35] [Q&A] [answered]
Hi, I'm trying to implement a chat into an Application and I wanted to use this client, but I can't even build the client after downloading because of a missing docker env. is there a Doc or Tutorial for this client?

**Comment by reda-alaoui** [comment, marked as answer]
The doc is [here](https://github.com/Cosium/matrix-communication-client) .
Just pull the client as a [dependency](https://github.com/Cosium/matrix-communication-client#dependency), no need to build the client.

**Reply by dhs-rec** [reply]
> The doc is [here](https://github.com/Cosium/matrix-communication-client) 

Sorry, but this is NOT documentation. That's just two little examples. There's no list of supported features, or other things that belong into a good documentation. [This](https://matrix-nio.readthedocs.io/en/latest/#) is documentation.

**Reply by reda-alaoui** [reply]
@dhs-rec , I think I don't work for you. If I do, where is my paycheck?
You can take your crappy attitude elsewhere.
